anchore_engine/common/helpers.py

```diff
@@ -366,3 +360,12 @@ def make_eval_record(userId, evalId, policyId, imageDigest, tag, final_action, e
     payload["last_updated"] = payload['created_at']
 
     return payload
+
+def safe_extract_json_value(value):
+    # support the legacy serialized json string
+    if isinstance(value, str):
```

Comment:
I don't think that is necessary to check for an instance of `str`. The `try/except` block can already handle this